datastore/query.go

```diff
@@ -808,21 +810,32 @@ func (c *Client) RunAggregationQuery(ctx context.Context, aq *AggregationQuery)
 	return ar, nil
 }
 
-// parseReadOptions translates Query read options into protobuf format.
-func parseReadOptions(q *Query) (*pb.ReadOptions, error) {
-	if t := q.trans; t != nil {
+func validateReadOptions(eventual bool, t *Transaction) error {
+	if t != nil {
```

Comment:
You might try reversing this so the "happy path" isn't indented.

``` golang
if t == nil {
  return nil
}

// t != nil
// ...
```

Reply:
https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow

Reply:
Thanks. Updated.